Leanplum/Cache/LPCache.m

```diff
@@ -41,69 +41,7 @@ - (void)setFileAttributes:(NSArray<LPFileAttribute *> *)fileAttributes {
     return regions;
 }
 
-//ToDo: Implement feature based
```

Comment:
let's put unrelated changes in separate PRs, to help better track where test coverage is headed